admin/app/templates/components/certification-session-report.hbs

```diff
@@ -0,0 +1,205 @@
+{{#bs-modal onHide=(action hide) open=show size="lg" as |modal|}}
+  {{#modal.header}}
+    <h4 class="modal-title">Analyse du PV de session</h4>
+  {{/modal.header}}
+
+  {{#modal.body}}
+    <div class="certification-session-report__body">
+
+      <section class="data-section data-section--listed-candidates">
+        <div class="row mb-2">
+          <div class="col-8">
+            <button class="data-section__switch" type="button" {{action "toggleSection" "displayListedCandidatesSection"}}>Nombre de candidats trouvés</button>
+          </div>
+          <div class="col-3">
+            <span class="data-section__counter">{{candidates.length}}</span>
+          </div>
+          <div class="col-1">
+            {{#if candidates.length}}
+              {{fa-icon "check" class="data-section__status--success"}}
+            {{else}}
+              {{fa-icon "times" class="data-section__status--failure"}}
+            {{/if}}
+          </div>
+        </div>
+        {{#if (and candidates.length displayListedCandidatesSection)}}
```

Comment:
De manière générale c'est pas très "Ember way" de mettre de la logique 'and, or, not, ...' dans le hbs, il vaut mieux privilégier des computed properties.